llvm/include/llvm/Transforms/Utils/Local.h

```diff
@@ -515,6 +515,14 @@ Value *invertCondition(Value *Condition);
 /// function, explicitly materialize the maximal set in the IR.
 bool inferAttributesFromOthers(Function &F);
 
+///===---------------------------------------------------------------------===//
+///  DebugInfo utilities
```

Comment:
I would think it fits in with the existing "Dbg Intrinsic utilities" section:

https://github.com/llvm/llvm-project/blob/df8e0d057d67a8167f2bdc81d8b61033beb55da5/llvm/include/llvm/Transforms/Utils/Local.h#L258

Perhaps it's better to append it to the end of that section instead of creating a new, similar section?